src/main/webapp/join-room.html

```diff
@@ -3,6 +3,13 @@
   <head>
     <meta charset="UTF-8">
     <title>Join Youtube Party Room</title>
+    <style>
+      body{
+          margin: auto;
+          text-align: center;
+      }
+    </style>
+
   </head>
   <body>
     <div id="join-room-box">
```

Comment:
Delete the whole target folder. That's binary files. Shouldn't be included!

Reply:
The target directory cannot be deleted and still run on localhost. I put in a .gitignore to stop adding target to all future commits, though.